src/PushThread.cpp

```diff
@@ -1098,10 +1091,11 @@ void *PushThread::run_undetached(void *ptr)
 					try
 					{
 						//	Send it to DB
-						/*int ret =*/ shared->mdb->insert_param_Attr(cmd->ev_data_param, cmd->ev_data_type);
+						shared->mdb->insert_param_Attr(cmd->ev_data_param, cmd->ev_data_type);
 					}
 					catch(Tango::DevFailed  &e)
 					{
+						shared->set_nok_db(cmd->ev_data->attr_name, string(e.errors[0].desc));
```

Comment:
I removed the shared->set_nok_db() and replaced it with an error message. This required we derive the PushThread from the LogAdaptor.